ts/components/CompositionInput.tsx

```diff
@@ -562,6 +562,31 @@ export function CompositionInput(props: Props): React.ReactElement {
   const callbacksRef = React.useRef(unstaleCallbacks);
   callbacksRef.current = unstaleCallbacks;
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const bindingVar: any = {
+    onShortKeyEnter: {
+      key: 13, // 13 = Enter
+      shortKey: true,
+      handler: () => callbacksRef.current.onShortKeyEnter(),
+    },
+    onEscape: {
+      key: 27,
+      handler: () => callbacksRef.current.onEscape(),
+    }, // 27 = Escape
+    onBackspace: {
+      key: 8,
+      handler: () => callbacksRef.current.onBackspace(),
+    }, // 8 = Backspace
+  };
```

Comment:
Moving it outside memo will make a performance a bit worst. Also can you use use a proper type instead of any?